src/TimeGrid.js

```diff
@@ -64,28 +60,22 @@ export default class TimeGrid extends Component {
     }
   }
 
-  componentDidUpdate() {
-    if (this.props.width == null) {
-      this.measureGutter()
-    }
-
-    this.applyScroll()
-    //this.checkOverflow()
-  }
-
-  UNSAFE_componentWillReceiveProps(nextProps) {
+  componentDidUpdate(prevProps) {
```

Comment:
some of this should be in a getSnapshotBeforeUpdate hook, it's important that the measuring code run _before_ render